<script>
    const romevingGarbage = ()=>{
       console.log("removingGarbage Triggered...."); 
       return new Promise((resolve,reject)=>{
          setTimeout(()=>{
              //resolve("Garbage Removed.....");
              reject("Garbage Rejected....");
          },3000);
        }); 
    }
    
    const roomCleaning = ()=>{
        console.log("roomCleaning Triggered......");
        return new Promise((resolve,reject)=>{
            setTimeout(()=>{
                //resolve("Room Cleaned...");
               reject("Room Cleaning Rejected......");
            },2000);
        });
    }
    
    /*
      We have multiple asynchronous operation.
    */
    
    Promise.all([romevingGarbage(),roomCleaning()])
    .then((results)=>{
      console.log("Promise is resolved......");  
      console.log(results[0]);
      console.log(results[1]);
    })
    .catch((rejectionResult)=>{
        console.log(rejectionResult);
    });
    </script>